Kernel/Bus/PCI/Definitions.h

```diff
@@ -260,7 +260,14 @@ class Capability {
 };
 
 AK_TYPEDEF_DISTINCT_ORDERED_ID(u8, ClassCode);
+AK_MAKE_COMPARABLE_TO_ENUM(ClassCode, ClassID)
+
 AK_TYPEDEF_DISTINCT_ORDERED_ID(u8, SubclassCode);
```

Comment:
I'm suspicious of this. Should we have this SubclassCode instead be a union of these enum types, rather than a distinct ordered id? It doesn't feel like it fits with the same semantics of other ordered IDs, like pid/gid/uid

Reply:
Well, in some way, PCI class codes are defined (by the PCI-SIG) as ordered IDs, so this is why I chose this macro.

Reply:
Eh. If we want to keep these as distinct ordered ids rather than just raw u8 or as `struct Foo { u8 value; explicit u8 operator u8() { return value; }` type nonsense, then the new macro should at least be named so that it's obvious it's for DistinctNumerics only

Reply:
I hope the new name of the macro is long enough